openapi-v2.yaml

```diff
@@ -1347,8 +1347,50 @@ definitions:
         type: integer
         format: uint64
 
+  AssetActivityLog:
+    type: object
+    properties:
+      id:
+        type: string
+        description: The ID of the activity
+      event_at:
+        description: time event took place (RFC3339)
+        type: string
+        format: date-time
+      action:
+        type: string
+        description: type of the event
+      username:
+        description: User who performed action
+        type: string
+      asset:
+        description: The asset details
+        type: object
+        properties:
+          id:
+            description: The asset ID
+            type: string
+          name:
+            description: The asset name
+            type: string
+          namespace:
+            description: The namespace that the asset belongs to
+            type: string
+          asset_type:
+            description: The type of the asset
+            $ref: "#/definitions/AssetType"
+          array_task_id:
```

Comment:
NIT: `array_task_id` shouldn't be inside the `asset`.

Reply:
`array_task_id` is only for arrays (that's why i named it at first task_id as a more generic name). 
Since it is `array_task_id` makes more sense to me to be inside asset where asset_type is defined.
But not a strong opinion, it does not make much of a difference.

Do you prefer to have it inside the `AssetActivityLog` @SarantopoulosKon ?

Reply:
@disideris personally i'd prefer it inside `AssetActivityLog`, `array_task_id` is a reference to an array task not to an `Asset`.

Furthermore, `Asset`  model is used elsewhere, i'd prefer to have it as close (even if it's partial) to the `Asset` model we use in asset listings.